api/cpp/docs/cmake_reference.md

```diff
@@ -82,3 +82,9 @@ In the following example, the translation files will be bundled from `lang/<lang
 ```cmake
 set_property(TARGET my_application PROPERTY SLINT_BUNDLE_TRANSLATIONS "${CMAKE_CURRENT_SOURCE_DIR}/lang")
 ```
+
+## Translation Domain
+
+By default, the domain used for translations is the name of the CMake target the `.slint` files are targeted with.
+Use the `SLINT_TRANSLATION_DOMAIN` target property to override this and use the specified value as domain, instead.
+This is useful in build environments where the target name is given and not suitable, such as esp-idf.
```

Comment:
This is useful in many other case. Not sure if mentioning esp-idf here is worth it. But I guess it doesn't hurt